Vizzuality/foodscapes
# [FOOD-62] initial Docker/Docker Compose setup for TiTiler

This is just a small change over Biel's TiTiler app and container setup, plus the beginning of Getting Started docs in the main `README`.

## How to test

- place the current `foodscapes_stack_cog_lzw.tif` in `tiler/data/foodscapes_stack_cog_lzw.tif` in the repo
- follow the _Quick start_ in the root `README` (paying attention to creating a `.env` file in the repo root from the `env.default` supplied in this PR's branch)
- the tiler service should spin up and be available on http://localhost:3201 (or whatever port is configured in `.env`)
